packages/blocks/src/components/DefaultBlock.tsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable prettier/prettier */
+import React from 'react';
+import { EditorBlock } from './WordPressBlocksViewer.js';
+
+/**
+ * DefaultBlock is an instance of WordPressBlock that is used in case no matching component is found when rendering the list of blocks.
+ * @param param0
+ * @returns JSX.Element
+ */
+export default function DefaultBlock({ renderedHtml }: EditorBlock) {
+  // eslint-disable-next-line react/no-danger
+  return <span dangerouslySetInnerHTML={{ __html: renderedHtml ?? '' }} />;
```

Comment:
I'm curious if there is any reprocussions of using a `span` element here. For exmaple, if a site has global css that effects spans. Could be rare, but still.

Not sure if it's something we need to address, but just writing here now in case it crops up in the future we have something to reference.

Reply:
Either a `div` or a `span` is valid here. I just wanted an inline element.